include/mbgl/util/projection.hpp

```diff
@@ -13,7 +13,7 @@ namespace mbgl {
 class Projection {
 public:
     // Map pixel width at given scale.
-    static double worldSize(double scale) {
+    static constexpr double worldSize(double scale) {
```

Comment:
Is there a need for `TransformState` to be `constexpr`? Based on the history with GCC 4.9 it sounded like something we should avoid using unless it's truly necessary.